mysql-plugin/src/e2e-test/features/mysqlsource/DesignTime.feature

```diff
@@ -0,0 +1,82 @@
+# Copyright © 2022 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License..
+
+Feature: MySQL Source - Design time scenarios
+
+  @TS-MYSQL-SOURCE-DSGN-01 @MYSQL_SOURCE_TEST
+  Scenario: Verify user can validate successfully when plugin is configured for table with connection and basic details
```

Comment:
We can rephrase the above scenario to be more understandable/readable -  Verify user is able to validate the plugin when configured for basic Section.

Reply:
done